Apps/WebClient/src/ClientApp/app/components/modal/covid.vue

```diff
@@ -7,6 +7,7 @@
 <template>
   <b-modal
     id="covid-modal"
+    v-model="isVisible"
```

Comment:
So this sets the visibility? I like it! guess that came with the latest package update